src/App.jsx

```diff
@@ -14,57 +15,62 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedItem, setSelectedItem] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleItemClick = good => {
+    if (selectedItem === good) {
+      setSelectedItem(null);
+    } else {
+      setSelectedItem(good);
+    }
```

Comment:
use functional setState for it 
```suggestion
    setSelectedItem((prevState) => (
      prevState === good 
        ? null
        : good
    ));
```